src/functionalTest/java/uk/gov/hmcts/reform/em/hrs/BaseTest.java

```diff
@@ -279,4 +283,40 @@ protected String filename(String caseRef) {
         return FOLDER + "/" + JURISDICTION + "-" + LOCATION_CODE + "-" + caseRef + "_" + TIME
             + "-UTC_" + SEGMENT + ".mp4";
     }
+
+    public String closeCase(final String caseRef) {
+
+        String s2sToken = extendedCcdHelper.getCcdS2sToken();
+        String userToken = idamClient.getAccessToken(HRS_TESTER, "4590fgvhbfgbDdffm3lk4j");
+        String uid = idamClient.getUserInfo(userToken).getUid();
+
+        final Optional<CaseDetails> optionalCaseDetails = searchForCase(caseRef);
```

Comment:
maybe we can pass the caseId as well as the caseRef so that we don't have to search for the case again